general/CatalogoDatiInformativiPerPsP.xsd

```diff
@@ -0,0 +1,552 @@
+  </xs:simpleType>
+
+  <xs:simpleType name="stText140">
+    <xs:restriction base="xs:string">
+      <xs:minLength value="1" />
+      <xs:maxLength value="140" />
+    </xs:restriction>
+  </xs:simpleType>
+
+  <xs:simpleType name="stIdentificativoIntermediario">
+    <xs:annotation>
+      <xs:documentation>
+				Identificativo dell'intermediario, identificato tramite il suo codice fiscale
+		</xs:documentation>
+    </xs:annotation>
+    <xs:restriction base="xs:string">
+      <xs:pattern value="[0-9]{11}"></xs:pattern>
+    </xs:restriction>
+  </xs:simpleType>
+
+  <xs:simpleType name="stIdentificativoCanale">
+    <xs:annotation>
+      <xs:documentation>
+				formato di identificazione del canale composto da codice fiscale (11 cifre ) seguito da una tassonomia definita come: 
+
+					- WEB : rappresenta un servizio offerto on-line 
+					- ATM : rappresenta un servizio offerto tramite sportello automatico
+					- APP : rappresenta un servizio offerto tramite applicazione mobile
+					- PAGOPA : rappresenta un servizio di pagamento non carte integrato all'interno del  WISP / appIO
+					- ON_US : rappresenta un servizio di pagamento come acquirer dedicato alle transazioni delle proprie carte
+					- CARD : rappresente un servizio di pagamento come acquirer dedicato alle transazioni per le carte non emesse dal PSP
+
+					Per retrocompatibilità vengono ammessi anche codici numerici a 2 cifre dopo il codice fiscale.
+			</xs:documentation>
+    </xs:annotation>
+    <xs:restriction base="xs:string">
+      <xs:pattern value="[0-9]{11}_([0-9]{2})?(_)?(WEB|ATM|APP|PAGOPA|ONUS|CARD)?"></xs:pattern>
+    </xs:restriction>
+  </xs:simpleType>
+
+  <xs:simpleType name="stImporto">
+    <xs:restriction base="xs:decimal">
+      <xs:minInclusive value="0.01" />
+      <xs:fractionDigits value="2" />
+      <xs:totalDigits value="18" />
+      <xs:pattern value="\d+\.\d{2}" />
+    </xs:restriction>
+  </xs:simpleType>
+
+  <xs:simpleType name="stImportoZero">
+    <xs:restriction base="xs:decimal">
+      <xs:minInclusive value="0.00" />
+      <xs:fractionDigits value="2" />
+      <xs:totalDigits value="18" />
+      <xs:pattern value="\d+\.\d{2}" />
+    </xs:restriction>
+  </xs:simpleType>
+
+  <xs:simpleType name="stImportoMassimoFascia">
+    <xs:restriction base="xs:decimal">
+      <xs:enumeration value="5.00"></xs:enumeration>
+      <xs:enumeration value="10.00"></xs:enumeration>
+      <xs:enumeration value="30.00"></xs:enumeration>
+
+      <xs:enumeration value="50.00"></xs:enumeration>
+      <xs:enumeration value="100.00"></xs:enumeration>
+
+      <xs:enumeration value="150.00"></xs:enumeration>
+      <xs:enumeration value="200.00"></xs:enumeration>
+
+      <xs:enumeration value="250.00"></xs:enumeration>
+      <xs:enumeration value="500.00"></xs:enumeration>
+      <xs:enumeration value="1000.00"></xs:enumeration>
+      <xs:enumeration value="1500.00"></xs:enumeration>
+      <xs:enumeration value="2000.00"></xs:enumeration>
```

Comment:
NOTA: il vincolo sull'importoMassimoFascia vincola anche la disponibilità del PSP nel gestire i pagamenti modello 1 (es: attualmente AMEX ha un massimale di 9000 euro, con questa modifica AMEX dovrebbe abbassare il massimale a 2000 euro >> il PSP sceglie quanto è disposto a gestire sul WISP2 grazie a questo campo e, in più, i pagamenti con carta sono vincolati all'importo massimo gestibile su vpos, quindi il PSP non può selezionare 999999999.99 come importo massimo di default, per non dare informazioni fuorvianti all'utente (che vedrebbe il PSP disponibile ma non potrebbe pagare) e per non trovarsi a gestire importi non voluti (es: 10000 euro)

Reply:
Si @furlaniilaria il problema che descrivi è stato riportato anche [qui](https://pagopa.atlassian.net/wiki/spaces/~345445188/pages/106889702/Modifiche+Catalogo+Dati+Informativi#Issue%231) per completezza d'analisi.